tests/testPosixStorage.py

```diff
@@ -139,7 +139,14 @@ def testAbsolutePath(self):
         f.close()
         del f
         f = storage.getLocalFile('foo.txt')
-        self.assertIsInstance(f, file)
+        try:
+            fileType = file
+        except NameError:
+            from io import IOBase
+            fileType = IOBase
```

Comment:
I'm curious whether it matters that `file` inherits from `io.IOBase`. I realize that it may mean a tiny change to one test, but is that a bad thing?